javascript/javascript2/week1/spirit-animal-name-generator.js

```diff
@@ -0,0 +1,74 @@
+//Spirit animal name generator
+const spiritAnimal = ["The Lion", "The Cheetah", "The Horse", "The Rabbit", "The Elephant", "The Camel", "The Panda", "The Dog", "The Deer", "The Cat"]
+console.log(spiritAnimal.length)
+
+const yourSpiritAnimal = [
```

Comment:
This would be nicer as an object instead of an array of objects. You can have spaces in keys if you use this (for example): `spiritAnimalObject['The Lion'] = "Natural-born leader,  courageous,  powerful"`

Reply:
Used object instead of an array of objects and made necessary changes.